front/src/applications/stdcm/components/StdcmWarningBox.tsx

```diff
@@ -25,8 +27,16 @@ const StdcmWarningBox = ({
   removeDestinationArrivalTime,
 }: StdcmWarningBoxProps) => {
   const { t } = useTranslation('stdcm');
+  const ref = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (ref.current) {
+      ref.current.scrollIntoView({ behavior: 'smooth' });
+    }
+  });
```

Comment:
Shouldn't we add this ?

```suggestion
  useEffect(() => {
    if (ref.current) {
      ref.current.scrollIntoView({ behavior: 'smooth' });
    }
  }, []);
```

Reply:
Not really but we need to add `errorType` and `errorDetails` in dependencies